.github/workflows/tex-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
 
         # Test python 3.6, 3.7, and 3.8 by default.
```

Comment:
update the comment to test python 3.8, 3.9, 3.10 by default.

Reply:
Fixed.